partition/androidboot.go

```diff
@@ -75,3 +76,13 @@ func (a *androidboot) SetBootVars(values map[string]string) error {
 	}
 	return env.Save()
 }
+
+func (a *androidboot) Reboot() error {
```

Comment:
This one is tricky as it is writing in a global folder (/run/systemd) for which root permissions are needed, and I do not see an equivalent of MockCommand for this.